neutron/assetlist.json

```diff
@@ -2142,12 +2143,23 @@
       "display": "maxBTC",
       "symbol": "maxBTC",
       "logo_URIs": {
-        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/maxBTC.svg"
```

Comment:
The overall change to the rounded logo is good, but the old image (maxBTC (with the uppercase BTC)) will need to be removed, because we cannot have unused images.